service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProvider.java

```diff
@@ -57,4 +75,14 @@ protected CaseManagementOrderEmailContentProvider(@Value("${ccd.ui.base.url}") S
                 uiBaseUrl, JURISDICTION, CASE_TYPE, caseDetails.getId())
         );
     }
+
+    private Map<String, Object> buildCommonCMODocumentNotificationParameters(final CaseDetails caseDetails,
```

Comment:
How is that common if it is used only once?

Reply:
the plan is it will be reused once notifications to representatives/respondents is added.